waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -98,8 +98,8 @@ proc init*(T: type Cheque, buffer: seq[byte]): ProtoResult[T] =
 
 
 # TODO Assume we calculated cheque
-proc sendCheque*(ws: WakuSwap) {.async.} =
-  let peerOpt = ws.peerManager.selectPeer(WakuSwapCodec)
+proc sendCheque*(ws: WakuSwap, peerId : PeerId) {.async.} =
```

Comment:
See my comment elsewhere on changing this to `peerInfo`. It would require changing the `credit` and `debit` procedures to take `peerInfo` argument as well. `peerInfo` should be available on the connection when calling `credit`/`debit` from `waku_store`. The `getPeerInfo()` procedure in the manager then becomes obsolete.

Reply:
Right. I'll make the necessary changes now. Thanks